bees/prometheusbee/prometheusbee.go

```diff
@@ -0,0 +1,253 @@
+/*
+ *    Copyright (C) 2019 	  CalmBit
```

Comment:
Again, unbelievably nitpicky, but I'd try to keep the names left-aligned here.

Reply:
Not nitpicky! I just thought it'd be better right-aligned, but I'll fix it :)